tests/vic-machine.bats

```diff
@@ -0,0 +1,130 @@
+# Copyright 2016 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#!/usr/bin/env bats
+
+load helpers/helpers
+load 'helpers/bats-support/load'
+load 'helpers/bats-assert/load'
+
+installer="/sbin/vic-machine"
+docker="/usr/bin/docker"
+log="/sbin"
+
+setup () {
+    assert [ -x "${installer}" ]
+    assert [ -x "$docker" ]
+
+    export GOVC_URL=$VIC_ESX_TEST_URL
+    export GOVC_INSECURE=true
+    export GOVC_PERSIST_SESSION=true
+
+    datacenter=`govc ls -k / | awk -F/ '{print $2}'`
+    echo $datacenter
+    host=`govc ls -k /${datacenter}/host/`
+    host=`echo ${host} | awk '{print $1}'`
+    echo $host
+    POOL_PATH=`govc ls $host/Resources`
+    echo $POOL_PATH
+    datastore=`govc ls /$datacenter/datastore | awk -F/ '{print $4}'`
+    IMAGE_STORE_NAME=`echo $datastore | awk '{print $1}'`
+    echo $IMAGE_STORE_NAME
+    USER=$(awk -F: '{print $1}' <<<"$GOVC_URL")
+    echo $USER
+    passurl=$(awk -F: '{print $2}' <<<"$GOVC_URL")
+    echo $passurl
+    PASSWORD=$(awk -F@ '{print $1}' <<<"$passurl")
+    echo $PASSWORD
```

Comment:
I think this is ok as long as access to ci.vmware.run is restricted, but this and other places credentials are echoed would have to be changed if we opened up access to CI.